components/Modal/CreateBlueprint.js

```diff
@@ -92,216 +114,151 @@ class CreateBlueprint extends React.Component {
   handleErrorDuplicate(blueprintName) {
     const nameNoSpaces = blueprintName.replace(/\s+/g, "-");
     if (this.props.blueprintNames.includes(nameNoSpaces)) {
-      this.props.setModalCreateBlueprintErrorDuplicateVisible(true);
+      this.setState({ errorNameDuplicate: true });
     }
   }
 
   handleErrorName(blueprintName) {
-    if (blueprintName === "" && this.props.createBlueprint.checkErrors) {
+    if (blueprintName === "" && this.state.checkErrors) {
       setTimeout(() => {
-        this.props.setModalCreateBlueprintErrorNameVisible(true);
+        this.setState({ errorNameEmpty: true });
       }, 200);
     }
   }
 
-  showInlineError() {
-    this.props.setModalCreateBlueprintErrorInline(true);
-  }
-
   render() {
-    const { createBlueprint } = this.props;
     return (
-      <div
-        className="modal fade"
-        id="cmpsr-modal-crt-blueprint"
-        tabIndex="-1"
-        role="dialog"
-        aria-labelledby="myModalLabel"
-        aria-hidden="true"
-      >
-        <div className="modal-dialog">
-          <div className="modal-content">
-            <div className="modal-header">
-              <button
-                type="button"
-                className="close"
-                data-dismiss="modal"
-                aria-hidden="true"
-                onMouseEnter={() => this.errorChecking(false)}
-                onMouseLeave={() => this.errorChecking(true)}
-                onClick={e => this.dismissErrors(e)}
-              >
-                <span className="pficon pficon-close" />
-              </button>
-              <h4 className="modal-title" id="myModalLabel">
-                <FormattedMessage defaultMessage="Create Blueprint" />
-              </h4>
+      <Modal show id="cmpsr-modal-crt-blueprint">
+        <Modal.Header>
+          <Modal.CloseButton onClick={this.props.close} />
+          <FormattedMessage defaultMessage="Create Blueprint" />
```

Comment:
Weird, I posted a comment here, and it disappeared.

The font weight isn't as heavy as it should be. I think it's because you're missing `<Modal.Title>` here, which is being used in the [PF3 component example](https://patternfly-react.surge.sh/patternfly-3/?selectedKind=patternfly-react%2FForms%20and%20Controls%2FModal%20Overlay%2FComponents&selectedStory=Modal&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs).

Reply:
This is also the case with the Create Image modal.

Reply:
Right, good catch, I will add it back.